samples/gpio_interrupt/main.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * This is a basic sample of using the UART module. The program provides a
+ * basic echo functionality where the uart will write back whatever the user
+ * enters.
+ */
+#include <EVT/io/manager.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/io/UART.hpp>
+
+namespace IO = EVT::core::IO;
+
+constexpr int BAUD_RATE = 115200;
```

Comment:
Good call.  How do you feel about us having some common variables/constants file that holds the common BAUD_RATE and other variables across our projects.  That way we can easily enforce a standard within a project?

Reply:
I'm on the fence, on one hand I love that it would create unity, on the other hand I like transparency in the samples since it makes it easier for people just starting out on the team (don't have to do a bunch of clicking around to find out why things are the way that they are). I could really be persuaded either way.

Reply:
I see your point and I agree.